app/assets/javascripts/blacklight/hierarchy/hierarchy.js

```diff
@@ -2,30 +2,32 @@ Blacklight.onLoad(function(){
   Blacklight.do_hierarchical_facet_expand_contract_behavior();
 });
 
-(function($) {
-  Blacklight.do_hierarchical_facet_expand_contract_behavior = function() {
-    $( Blacklight.do_hierarchical_facet_expand_contract_behavior.selector ).each (
-        Blacklight.hierarchical_facet_expand_contract
-     );
+(function() {
```

Comment:
Can an arrow function expression be used here too, per below? Or is there a reason to prefer the function keyword in this context?